monitoring/rid_qualifier/create_flight_record_from_kml.py

```diff
@@ -48,23 +42,36 @@ def check_if_vertex_is_correct(point1, point2, point3, flight_distance):
         ) < 0.2, f'Generated vertex is not correct. {point1}, {point2}, {point3}, {flight_distance}'
 
 
+def get_track_angle(point1, point2):
+    x1, y1 = point1
+    x2, y2 = point2
+    if x2 == x1:
+        return 0
+    else:
+        return math.degrees(math.atan((y2 -y1)/(x2 - x1)))
```

Comment:
Looks like this isn't quite what we want.  A north track should be 0, east should be 90, south 180, and west 270 ("degrees east of north").  An increase in `x` here should indicate that longitude is increasing (an eastward movement) and an increase in `y` should indicate that latitude is increasing (a northward movement).  Let's at least test all the cardinal directions (N, E, S, W).

Also, we'll sometimes get no movement in the denominator (regardless of whether it's `x` or `y`) and we still want this to compute correctly.  So, let's use [`atan2`](https://docs.python.org/3/library/math.html#math.atan2).  Since this might be tricky to ensure we get right, we can add a unit test that verifies we obtain known angles with known `point1` and `point2`.  For instance, if `point2.x` = `point1.x` and `point2.y` = `point2.y + <something positive>`, then we know the track should be `0` (north).

Reply:
Updated with supported use-cases.